terraform/services/snyk-integration/main.tf

```
@@ -0,0 +1,67 @@
+data "aws_iam_policy" "developer_boundary_policy" {
+  name = "developer-boundary-policy"
+}
+
+data "aws_ssm_parameter" "external_id" {
+  name = "/snyk-integration/external_id"
```

Use dashes in parameter names

```suggestion
  name = "/snyk-integration/external-id"
```